design-system/src/theme-provider.visualspec.js

```diff
@@ -8,30 +8,30 @@ describe('ThemeProvider', () => {
     await percySnapshot(page, 'ThemeProvider');
   });
 });
-describe('Interactive', () => {
-  it('applies changes to global and local theme provider', async () => {
-    await page.goto(`${globalThis.HOST}/theme-provider/interactive`);
-    const doc = await getDocument(page);
+// describe('Interactive', () => {
+//   it('applies changes to global and local theme provider', async () => {
+//     await page.goto(`${globalThis.HOST}/theme-provider/interactive`);
+//     const doc = await getDocument(page);
 
-    // change global theme
-    const globalThemeChangeButton = await queries.findByText(
-      doc,
-      'change global theme'
-    );
+//     // change global theme
+//     const globalThemeChangeButton = await queries.findByText(
+//       doc,
+//       'change global theme'
+//     );
 
-    await globalThemeChangeButton.click();
-    await page.waitForSelector('[data-theme="recolouring"]');
-    // TODO: uncomment when issue with Percy is resolved
-    // await percySnapshot(page, 'ThemeProvider - after global theme change');
+//     await globalThemeChangeButton.click();
+//     await page.waitForSelector('[data-theme="default"]');
+//     // TODO: uncomment when issue with Percy is resolved
+//     // await percySnapshot(page, 'ThemeProvider - after global theme change');
 
-    // change local theme
-    const localThemeChangeButton = await queries.findByText(
-      doc,
-      'change local theme'
-    );
-    await localThemeChangeButton.click();
-    await page.waitForSelector('[data-theme="recolouring"]');
-    // TODO: uncomment when issue with Percy is resolved
-    // await percySnapshot(page, 'ThemeProvider - after local theme change');
-  });
-});
+//     // change local theme
+//     const localThemeChangeButton = await queries.findByText(
+//       doc,
+//       'change local theme'
+//     );
+//     await localThemeChangeButton.click();
+//     await page.waitForSelector('[data-theme="default"]');
+//     // TODO: uncomment when issue with Percy is resolved
+//     // await percySnapshot(page, 'ThemeProvider - after local theme change');
+//   });
+// });
```

Comment:
Perhaps we could just delete that?

Reply:
We agreed to leave it commented out just so we have a sample of the test block in the event we introduce new themes later on.